local-environment/src/networks/local-env/configurations/cardano/config-pool1.json

```diff
@@ -54,25 +54,25 @@
   "TraceLocalErrorPolicy": true,
   "TraceLocalHandshake": true,
   "TraceLocalRootPeers": true,
-  "TraceLocalTxSubmissionProtocol": false,
-  "TraceLocalTxSubmissionServer": false,
+  "TraceLocalTxSubmissionProtocol": true,
+  "TraceLocalTxSubmissionServer": true,
   "TraceMempool": true,
   "TraceMux": false,
   "TracePeerSelection": true,
   "TracePeerSelectionActions": true,
   "TracePublicRootPeers": true,
   "TraceServer": true,
-  "TraceTxInbound": false,
-  "TraceTxOutbound": false,
-  "TraceTxSubmissionProtocol": false,
-  "TracingVerbosity": "NormalVerbosity",
+  "TraceTxInbound": true,
+  "TraceTxOutbound": true,
+  "TraceTxSubmissionProtocol": true,
+  "TracingVerbosity": "MaximalVerbosity",
```

Comment:
Is this something we want to wire up to a parameter we can override when kicking off the workflow?

Reply:
running things with debug cranked up can potentially create performance problems that aren't there otherwise...

Reply:
You are right, reverted.